packages/experimental/Button/src/Button.types.ts

```diff
@@ -114,6 +114,11 @@ export interface ButtonProps extends Omit<IWithPressableOptions<ViewProps>, 'onP
 
   /** A floating action button  */
   fab?: boolean;
+
+  /**
+   * Foreground color for the text and/or icon of the button
+   */
+  color?: ColorValue;
```

Comment:
Why is color being added as a prop on Button?

Reply:
Oh sorry, I just reread Ruriko's comment:

> We may need to change button to expose some of its tokens as props, such as the foreground color/font tokens.

Does adding color as a prop help setting the color of the text for button and have you tried it with the font styles as well?

Reply:
Not yet. I was trying to use props.color || tokens.color in Button.style, but it didn't work